app/lib/features/chat/widgets/rooms_list.dart

```diff
@@ -26,6 +29,7 @@ class RoomsListWidget extends ConsumerWidget {
     final showSearch = ref.watch(_searchToggleProvider);
     final searchNotifier = ref.watch(_searchToggleProvider.notifier);
     final inSideBar = ref.watch(inSideBarProvider);
+    final size = MediaQuery.of(ctx).size;
```

Comment:
This is a ```BuildContext``` of the Chat Wrapper or easily said ```Subshell``` which is taking care of the whole shell size change rather than list only. It'd be wrong to use ```RoomList``` display context here... 

I think we might want to remove that need of context and instead use shell keys context ???